### PR TITLE
Added possibility to use SAML authentication when ordering service

### PR DESCRIPTION
**Pull request by khalima**

### CODE REVIEW

File: conf/env_production/settings.env.php [hunk 1]
```diff
@@ -12,30 +12,32 @@
...
...Dnw+CU8+ivjw99OI2292cLVd1cFC Y6Wx1fBgJps+d85fWFsz9kppYeAv7PLbMEesj4OPXGYAF84bBmrdNwcl0SoAVfzZ WTAAJZdxGAKsSpEdokQu7gsAXegLTFGCrSkxI/9337ZeMLd6LJyrWYxXFmxk6R5u owmZwKHxXOpq37ZlJksnCAghvoNjqdA0WBQeXNCrwrScWIdQIWBJCLYj/pYIBckZ Axvjm21cAgLl98lnCtWG3HG2t4GykxCsgtdY8hPw0Ze/oXBZpTY78DKD0pOW6ndj WGgtjZWiW6yrj8mlV8hWqCULfS/rfwIDAQABo4IBbTCCAWkwHwYDVR0jBBgwFoAU W9CKHJoyW+C13ZZUG+GGKLD9tr0wHQYDVR0OBBYEFMqg7/cYdb2o9CSfG4wtgFrc gwH9MA4GA1UdDwEB/wQEAwIFoDAMBgNVHRMBAf8EAjAAMB0GA1UdJQQWMBQGCCsG AQUFBwMBBggrBgEFBQcDAjAiBgNVHSAEGzAZMA0GCysGAQQBsjEBAgIdMAgGBmeB DAECATA6BgNVHR8EMzAxMC+gLaArhilodHRwOi8vY3JsLnVzZXJ0cnVzdC5jb20v VEVSRU5BU1NMQ0EyLmNybDBsBggrBgEFBQcBAQRgMF4wNQYIKwYBBQUHMAKGKWh0 dHA6Ly9jcnQudXNlcnRydXN0LmNvbS9URVJFTkFTU0xDQTIuY3J0MCUGCCsGAQUF BzABhhlodHRwOi8vb2NzcC51c2VydHJ1c3QuY29tMBwGA1UdEQQVMBOCEWxvZ2lu LmhlbHNpbmtpLmZpMA0GCSqGSIb3DQEBCwUAA4IBAQCJ2I4mTHMYM2SZuAf9XI2x Lk/VLU8LqZOGc5yvQCdLG3iLz6xkUU/ESG4Nrwyl8MN8IB/ul3bbZVriZ+htb4R4 CmB44srsj10lKAHcCDYbxUNj+EbewMXkXqBybjkqlih5JjdlH1eA72M8aDvtHIi9 7YOfX0qkuYX1MC17UL+IlD1Ed5oSdckyPUCDrTksNmZ3AegQVJlDmk8KL7xXm6qZ 1JFj+dX1uM1cPa0sf0ZK6fI8aOhPhKz+usclFplwR7ohnEp60S3KgH9VbhHpjItj DjLzl9rFbErZuexB3N7XJhesfd5IwneNaJ2oVjJZB9gzknuG6bwrURgOA8sHcxV0';
```
Comment (back-2-95) [inline conf/env_production/settings.env.php]: This should be in /data/rds/shared/settings.local.php file and not in Git

File: public/modules/custom/hy_esb/templates/user-information.html.twig [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+  {% endif %}
+  {% if email %}
+    <div class="box-card__field icon--mail">
+      <a href="mailto:{{ email }}">{{ email }}</a>
+    </div>
+  {% endif %}
+  {% if group %}
+    <div class="box-card__field icon--units-and-faculties">
+      {{ group }}
```
Comment (back-2-95) [inline public/modules/custom/hy_esb/templates/user-information.html.twig]: I don't know if we should show this at this point. It's quite technical value...

Comment (khalima) [reply public/modules/custom/hy_esb/templates/user-information.html.twig]: @back-2-95 I'd rather hide it from the module than from the template... Should it be included in the service order post?

Comment (khalima) [reply public/modules/custom/hy_esb/templates/user-information.html.twig]: @back-2-95 See: https://github.com/UniversityofHelsinki/ResearchDataServiceChannel/pull/42/commits/a4ad446e7cd77d477a1b3366bc6de5512bb82f90

Comment (back-2-95) [reply public/modules/custom/hy_esb/templates/user-information.html.twig]: 👍

**Pull request merged**